support-frontend/assets/components/subscriptionCheckouts/summary.tsx

```diff
@@ -1,41 +1,113 @@
-import { Component } from 'react';
-import type { $Call } from 'utility-types';
-import type { ProductPrice } from 'helpers/productPrice/productPrices';
+import { useState } from 'react';
 import 'helpers/productPrice/productPrices';
-import type { GridImg } from 'components/gridImage/gridImage';
 import { PriceLabel } from 'components/priceLabel/priceLabel';
 import type { BillingPeriod } from 'helpers/productPrice/billingPeriods';
+import type { ProductPrice } from 'helpers/productPrice/productPrices';
 import 'components/gridImage/gridImage';
 import type { Promotion } from 'helpers/productPrice/promotions';
 import { getAppliedPromo, hasDiscount } from 'helpers/productPrice/promotions';
 import { GuardianWeekly } from 'helpers/productPrice/subscriptions';
 import type { SubscriptionProduct } from 'helpers/productPrice/subscriptions';
 import SvgDropdownArrowUp from './dropDownArrowUp.svg';
-import styles from './summary.module.scss';
+import moduleStyles from './summary.module.scss';
 
-type GridImageType = typeof import('components/gridImage/gridImage').default;
-// Types
-export type DataListItem = {
+// Type declarations
+type DataListItem = {
```

Comment:
minor: I think we've so far had a slight preference for `interface` when it's applicable